docs/user/DockerUsage.md

```diff
@@ -0,0 +1,107 @@
+
+### Cloud Authentication
+
+The docker container includes the necessary cloud clis for interacting with the cloud providers. See [ansible cloud authentication](AnsibleCloudAuthentication.md) when deploying to GCP with external postgres
+
+### Docker Volume Mounts
+
+All configs needed by ansible are also needed to be mounted into the docker container. In general any file/folder path set via an ansible flag are equivalent to the file/folder being mounted to the docker container at `/config/<lower_case_variable_name>`. 
+
+See [Docker Volume Mapping](DockerVolumeMounts.md) for the full list of docker volume mappings.
+
+Examples:
+
+- The ansible flag `-e KUBECONFIG` is equivalent to `--volume <desire_path>:/config/kubeconfig` when running the docker container
+- The ansible flag `-e JUMP_SVR_PRIVATE_KEY` is equivalent to `--volume <desire_path>:/config/jump_svr_private_key` when running the docker container
+- The ansible flag `-e V4_CFG_SITEDEFAULT` is equivalent to `--volume <desire_path>:/config/v4_cfg_sitedefault` when running the docker container
+
+Below are the only exceptions:
+
+| Ansible Flag | Docker Mount Path | Description | Required |
+| :--- | :--- | :--- | ---: |
+| -e BASE_DIR | `/data` | local folder in which all the generated files can be stored. If you do not wish to save the files, this can be omitted | false |
+| --vault-password-file | `/config/vault_password_file` | Full path to file containing the Ansible vault password | false |
+
+### Variable Definitions (ansible-vars.yaml) File
+
+Prepare your `ansible-vars.yaml` file, as described in [Customize Input Values](../../README.md#customize-input-values).
+
+## Running
+
+Declare the Actions and Task to be performed
+
+### Actions
+
+Actions are used to install or uninstall software. One must be set when running the playbook.
+
+| Name | Description |
+| :--- | ---: |
+| Install | Installs the stack required for the specified tasks |
+| Uninstall | Uninstalls the stack required for the specified tasks |
+
+### Tasks
+
+Any number of tasks can be run at the same time. An action can run against a single task or all tasks.
+
+| Name | Description |
+| :--- | :--- |
+| baseline | Installs cluster level tooling needed for all viya deployments. These may include, cert-manager, ingress-nginx, nfs-client-provisioners and more. |
+| viya | Deploys viya |
+| cluster-logging | Installs cluster-wide logging using the [viya4-monitoring-kubernetes](https://github.com/sassoftware/viya4-monitoring-kubernetes) project. |
+| cluster-monitoring | Installs cluster-wide monitoring using the [viya4-monitoring-kubernetes](https://github.com/sassoftware/viya4-monitoring-kubernetes) project. |
+| viya-monitoring | Installs viya namespace level monitoring using the [viya4-monitoring-kubernetes](https://github.com/sassoftware/viya4-monitoring-kubernetes) project. |
+
+### Examples
```

Comment:
I think it would be valuable to add an example such as the following:

"I want to modify a customization under site-config for an existing viya deployment and reapply the manifest. I wish to continue to use the same deployment assets rather than pull the latest copy. "

''''bash
  ansible-playbook \
   --volume <desired_path>:/config/v4_cfg_deployment_assets \
    -e CONFIG=$HOME/ansible-vars.yaml \
    -e TFSTATE=$HOME/viya4-iac-aws/terraform.tfstate \
    viya4-deployment --tags "viya,install"
''''

Reply:
updated